server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -578,11 +541,18 @@ def delete_model_endpoints_resources(
                         project=project_name
                     ),
                 )
-                tsdb_connector.delete_tsdb_resources()
-            except mlrun.errors.MLRunInvalidMMStoreType:
-                # TODO : delete from v3io store for BC (if not ce).
-                pass
 
+            except mlrun.errors.MLRunInvalidMMStoreType:
+                # TODO: delete in 1.9.0 - for BC trying to delete from v3io store
+                if not mlrun.mlconf.is_ce_mode():
+                    tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
+                        project=project_name,
+                        tsdb_connection_string="v3io",
+                    )
+                else:
+                    tsdb_connector = None
+            if tsdb_connector:
+                tsdb_connector.delete_tsdb_resources()
```

Comment:
It can be simplified:
```suggestion
            except mlrun.errors.MLRunInvalidMMStoreType:
                # TODO: delete in 1.9.0 - for BC trying to delete from v3io store
                if not mlrun.mlconf.is_ce_mode():
                    tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
                        project=project_name,
                        tsdb_connection_string="v3io",
                    ).delete_tsdb_resources()
```
And possibly a log in the `else` clause

Reply:
I call `tsdb_connector.delete_tsdb_resources ` only once in your suggestion I need to call it twice, so I will stay with my version. :)